Code/src/main/resources/catalog/catalog.xml

```diff
@@ -18,105 +35,584 @@
+        		fractional division.
+        	</function>
+        	<function name="recip" signature="Fractional a => a -> a">
+        		reciprocal division.
+        	</function>
+        	<function name="fromRational" signature="Fractional a => Rational -> a">
+        		Conversion from a Rational (that is Ratio Integer).
+        	</function>
+        	<function name="pi" signature="Floating a => a"></function>
+        	<function name="exp" signature="Floating a => a -> a"></function>
+        	<function name="log" signature="Floating a => a -> a"></function>
+        	<function name="sqrt" signature="Floating a => a -> a"></function>
+        	<function name="(**)" signature="Floating a => a -> a -> a"></function>
+        	<function name="logBase" signature="Floating a => a -> a -> a"></function>
+        	<function name="sin" signature="Floating a => a -> a"></function>
+        	<function name="cos" signature="Floating a => a -> a"></function>
+        	<function name="tan" signature="Floating a => a -> a"></function>
+        	<function name="asin" signature="Floating a => a -> a"></function>
+        	<function name="acos" signature="Floating a => a -> a"></function>
+        	<function name="atan" signature="Floating a => a -> a"></function>
+        	<function name="sinh" signature="Floating a => a -> a"></function>
+        	<function name="cosh" signature="Floating a => a -> a"></function>
+        	<function name="tanh" signature="Floating a => a -> a"></function>
+        	<function name="asinh" signature="Floating a => a -> a"></function>
+        	<function name="acosh" signature="Floating a => a -> a"></function>
+        	<function name="atanh" signature="Floating a => a -> a"></function>
+        	<function name="properFraction" signature="(RealFrac a, Integral b) => a -> (b, a)">
+        		The function properFraction takes a real fractional number x and returns a pair (n,f) such that x = n+f, and:
+        		- n is an integral number with the same sign as x; and
+        		- f is a fraction with the same type and sign as x; and with absolute value less than 1.
+        	</function>
+        	<function name="truncate" signature="(RealFrac a, Integral b) => a -> b">
+        		truncate x returns the integer nearest x between zero and x
+        	</function>
+        	<function name="round" signature="(RealFrac a, Integral b) => a -> b">
+        		round x returns the nearest integer to x; the even integer if x is equidistant between two integers.
+        	</function>
+        	<function name="ceiling" signature="(RealFrac a, Integral b) => a -> b">
+        		ceiling x returns the least integer not less than x.
+        	</function>
+        	<function name="floor" signature="(RealFrac a, Integral b) => a -> b">
+        		floor x returns the greatest integer not greater than x.
+        	</function>
+        	<function name="floatRadix" signature="RealFloat a => a -> Integer">
```

Comment:
Are the functions from here up to and including scaleFloat interesting enough to include here?